after/plugin/securemodelines.vim

```diff
@@ -57,7 +57,7 @@ fun! <SID>IsInList(list, i) abort
 endfun
 
 fun! <SID>DoOne(item) abort
-    let l:matches = matchlist(a:item, '^\([a-z]\+\)\%([-+^]\?=[a-zA-Z0-9_\-,.\[\]]\+\)\?$')
+    let l:matches = matchlist(a:item, '^\([a-z]\+\)\%([-+^]\?=[a-zA-Z0-9_(\-,.\[\]]\+\)\?$')
```

Comment:
I'm happy to accept this as is, but curious: why `(` but not `)` also?

I see this as practical for `foldmarker`, what else is the use case?

Reply:
I need only `(` for `'cinoptions'`, so added just that. I'm happy to add `)` as well, it actually appears in one of `'cinoptions'`'s values (lots of special symbols there).